app/presenters/tslr_claims_presenter.rb

```diff
@@ -0,0 +1,29 @@
+require "csv"
+
+class TslrClaimsPresenter < BasePresenter
+  attr_accessor :decorated_collection
+
+  def initialize(claims)
+    self.decorated_collection = claims.map { |v| TslrClaimPresenter.new(v) }
+    super(claims)
+  end
+
+  def csv_headers
+    TslrClaim.csv_fields.map { |h| translation_for_field(h) }
+  end
+
+  def csv_string
```

Comment:
Again, potential DOS if there are lots of submissions - memory usage will grow linearly with the number of submissions as the entire CSV data will be held in memory as a string. Can this class write to and return a `Tempfile` instead of a string so we keep memory usage to a fixed size?